web/template/clients.gotmpl

```diff
@@ -105,12 +105,17 @@
                                         {{ end }}
                                     </a>
                                     <span class="secondary court_ref">
-                                                    {{ if .CourtRef }}
-                                                        {{ .CourtRef }}
-                                                    {{ else }}
-                                                        -
-                                                    {{ end }}
-                                                </span>
+                                        {{ if .CourtRef }}
+                                            {{ .CourtRef }}
+                                        {{ else }}
+                                            -
+                                        {{ end }}
+                                    </span>
+                                    <span class="secondary rem-warning">
```

Comment:
I would wrap the if around the span tag and add a Cypress test to cover the conditional logic